### ISSUE TITLE
social sharing for claimed badges

### ISSUE THREAD

**Issue by stenington** [mozilla/openbadges-badges#38]
After #36 this will live on the show panel.

I think step 1 is figuring out what gets shared. I'd recommend for the first pass we strike a balance between sharing something with enough information that it's meaningful (i.e. show the metadata, maybe an indicator of ownership) but not getting too fancy with the social site integration (i.e. no deep facebook opengraph integration).

**Comment by xmatthewx** [comment]
Agreed on step one. I do think we can offer two distinct options: 
- show a baked badge with basic embed 
- simply share the news on twitter etc: "I earned the Long-titled Badge from Mozfest #OpenBadges." Maybe include a link to the issuer. 

This gives most every earner something to do that spreads the word on their new badge and openbadges.

**Comment by threeqube** [comment]
:+1: works for me. 

We should have in back of mind deeper social integration though.

**Comment by stenington** [comment]
An interesting option for the twitter etc. sharing could be to link to a page somewhere that unpacks all the metadata of the badge, including links to the issuer, criteria, evidence, etc. and displays it nicely.

**Comment by threeqube** [comment]
@stenington I like the idea in theory but I'm having a hard time visualizing this...

**Comment by emgollie** [comment]
@stenington I really like that! I think we would want to have the Twitter et al messaging correspond: "Unpack the Long-titled Badge I just earned from Mozfest #OpenBadges" or something that is action-oriented on the part of the consumer.

**Comment by carlacasilli** [comment]
@stenington would the content from that link exist in the backpack? Or is this pointing back to the issuer's site?

**Comment by stenington** [comment]
@carlacasilli Well ideally I'd like it to point to a 3rd party site that exists solely to take pointers to badges, find their metadata, and display that in a nice way. I think that lets it be the most reusable... sort of like an imgur.com for badges.

**Comment by iamjessklein** [comment]
So this is a personalized page - a la blahblahbla.com/jess/badgename that:
1. links to the criteria url
2. links to the issuer
3. displays the link to the evidence page for the badge - if made public(?)

correct?

**Comment by carlacasilli** [comment]
Sounds like a great idea. What do we do in the meantime while that site doesn't exist, @stenington ? And @iamjessklein I'm hoping for right now, the link to the criteria URL and the issuer are going to the same place. Once that gets divided up (and, no doubt it will), skeptics are going to get nervous. Also, the process becomes more complicated.

**Comment by threeqube** [comment]
@carlacasilli would you mind elaborating on this? Not sure I quite understand where the skeptics get nervous and the complication ensues.

**Comment by carlacasilli** [comment]
@threeqube If we have links that go to different places for one badge, it makes it that much more complicated for someone to validate a badge. It requires multiple checks to verify. Not a problem if this is all handled by orgs that do this for $$$ but for the casual user or small business owner, the requirements multiply and the process becomes complicated.

**Comment by threeqube** [comment]
@carlacasilli But according to @stenington above, "Well ideally I'd like it to point to a 3rd party site that exists solely to take pointers to badges, find their metadata, and display that in a nice way."

So we'll find this stuff and display it for the folks who are seeking this info, no?

**Comment by carlacasilli** [comment]
@threeqube Does that exist or are we creating it? Who is the 3rd party?

**Comment by threeqube** [comment]
@carlacasilli doesn't exist. We have to create something.

**Comment by iamjessklein** [comment]
I am curious if this:

"Well ideally I'd like it to point to a 3rd party site that exists solely to take pointers to badges, find their metadata, and display that in a nice way."

is related to the badge directory concept.
 http://jessicaklein.blogspot.com/2013/11/badge-directory-yelp-for-badges.html

**Comment by stenington** [comment]
I think we would build it. We've built something very much like what I'm thinking of in the badge bakery with the "unbake" functionality. The display is ugly right now; the metadata just gets dumped to the page. But from a technical standpoint it's basically the sort of thing I'm talking about, and we can certainly make a nicer display.

By "third party site" I didn't mean an actual 3rd party would build and host it, I meant the site is a 3rd party, i.e. not part of this codebase.

**Comment by carlacasilli** [comment]
aha! thanks for the clarification, @stenington! :poultry_leg:
